tools/backups2datalad/adataset.py

```diff
@@ -423,3 +448,16 @@ class DatasetStats:
 class AssetsState(BaseModel):
     PATH: ClassVar[Path] = Path(".dandi", "assets-state.json")
     timestamp: datetime
+
+
+class AnnexedFile(BaseModel):
+    backend: str
+    bytesize: int
+    # error-messages: list[str]
+    file: str
+    # hashdirlower: str
+    # hashdirmixed: str
+    # humansize: str
+    key: str
+    # keyname: str
+    # mtime: Literal["unknown"] | ???
```

Comment:
```suggestion
    file: str
    key: str
```
since those just copy pasted or you have some intention for them?

Reply:
They're just there as a full record of the `git-annex find --json` output fields, in case they're ever needed, so that I don't have to experiment with git-annex again.